ydb/core/tablet_flat/flat_part_iter_multi.h

```diff
@@ -690,6 +688,13 @@ namespace NTable {
                 Groups.emplace_back(Part->GetGroupIndex(groupId), groupId);
                 GroupRemap[group] = idx;
             }
+
+            size_t KeyCellDefaultsSize = KeyCellDefaults->Size();
+            Key.resize(KeyCellDefaultsSize);
+
+            for (size_t pos = 0; pos < KeyCellDefaultsSize; ++pos) {
```

Comment:
It should start from `Part->Scheme->Groups[0].ColsKeyData.size()` (data page key schema), otherwise it's just a waste of cpu. We usually have it equal to `KeyCellDefaultsSize`, i.e. all key columns are present in the sst. This is used to fill missing key columns with a default value when adding a new key column.

Reply:
Updated.